help/garage.rst

```diff
@@ -43,7 +43,7 @@ What kind of issues can we help with:
 
 Notes:
 
-* All garages are open to all people.
+* All garages are open to all researchers working in Aalto.
```

Comment:
Are we open for student questions as well?

Reply:
Short answer is no. I think the consensus was that in some cases they can be researchers also when they are doing course work closer to research (e.g. special assignments under PI/postdoc supervision). In general we won't help them with their homework :)  but I would say it is kind to direct them towards the right path (e.g. we can't help you with your python code, but here a course you can take by yourself)

Reply:
And often if a student has a triton account, then it is very likely doing some research or special assignment under research group supervision